examples/dashboard-grafana-crossplane-xmetrics.yaml

```diff
@@ -0,0 +1,768 @@
+                "range": false,
+                "refId": "A"
+              }
+            ],
+            "title": "Kubernetes Crossplane.io Object v1alpha1 Ready Time",
+            "transform": "timeseries_to_rows",
+            "transformations": [
+              {
+                "id": "organize",
+                "options": {
+                  "excludeByName": {
+                    "Value": true,
+                    "__name__": true,
+                    "app": true,
+                    "app_kubernetes_io_instance": true,
+                    "app_kubernetes_io_name": true,
+                    "instance": true,
+                    "job": true,
+                    "kubernetes_namespace": true,
+                    "kubernetes_pod_name": true,
+                    "node": true,
+                    "pod_template_hash": true
+                  },
+                  "indexByName": {
+                    "Time": 1,
+                    "Value": 12,
+                    "__name__": 2,
+                    "app": 3,
+                    "app_kubernetes_io_instance": 4,
+                    "app_kubernetes_io_name": 5,
+                    "instance": 6,
+                    "job": 7,
+                    "kubernetes_namespace": 8,
+                    "kubernetes_pod_name": 9,
+                    "name": 0,
+                    "node": 10,
+                    "pod_template_hash": 11
+                  },
+                  "renameByName": {}
+                }
+              }
+            ],
+            "type": "table-old"
+          }
+        ],
+        "schemaVersion": 37,
+        "style": "dark",
+        "tags": [],
+        "templating": {
+          "list": [
+            {
+              "current": {
+                "selected": false,
+                "text": "rds-test-postgres-01",
+                "value": "rds-test-postgres-01"
+              },
+              "datasource": {
+                "type": "prometheus",
+                "uid": "PBFA97CFB590B2093"
+              },
+              "definition": "kubernetes_crossplane_io_Object_v1alpha1_labels",
+              "description": "Claim Name",
+              "hide": 0,
+              "includeAll": false,
+              "label": "Claim Name",
+              "multi": false,
+              "name": "claim_name",
+              "options": [],
+              "query": {
+                "query": "kubernetes_crossplane_io_Object_v1alpha1_labels",
+                "refId": "StandardVariableQuery"
+              },
+              "refresh": 2,
+              "regex": "/label_crossplane_io_claim_name=\\\"([^\\\"]+)\\\"/",
+              "skipUrlSync": false,
+              "sort": 2,
+              "type": "query"
+            }
+          ]
+        },
+        "time": {
+          "from": "now-30m",
+          "to": "now"
+        },
+        "timepicker": {},
+        "timezone": "",
+        "title": "Crossplane X-Metrics",
+        "uid": "dt-jfvVIk",
```

Comment:
when using crossplane-provider-grafana please remove this line - we using external-name calculation for this